contrib/earthfile-syntax-highlighting/syntaxes/earthfile.tmLanguage.json

```diff
@@ -264,4 +272,4 @@
 			]
 		}
 	}
-}
+}
```

Comment:
nit: don't remove the final endline char, we aim to keep our files as POSIX files.